doc/en/monitoring/user-portal.rst

```diff
@@ -6,17 +6,17 @@ User portals
 Some clusters offer a portal to monitor your current jobs and even your
 recently completed jobs. Here they are:
 
-- **Béluga** : https://portail.beluga.calculquebec.ca
-- **Narval** : https://portail.narval.calculquebec.ca
-- **Niagara** : https://my.scinet.utoronto.ca
+- **Narval**: https://portail.narval.calculquebec.ca
+- **Rorqual**: https://metrix.rorqual.calculquebec.ca
+- **Trillium and Trillium GPU**: https://my.scinet.utoronto.ca
 
 In all cases, you must log in with your `CCDB <https://ccdb.alliancecan.ca>`__
 portal credentials.
 
```

Comment:
J’ajouterais cette note : « Following the 2025 infrastructure renewal, not all clusters have a user portal already. New portals and functionalities will be added progressively. »